hack/test-cmd.sh

```diff
@@ -417,6 +417,26 @@ for version in "${kube_api_versions[@]}"; do
   # Post-condition: no POD is running
   kube::test::get_object_assert pods "{{range.items}}{{$id_field}}:{{end}}" ''
 
+  ### Create two PODs from 1 yaml file
+  # Pre-condition: no POD is running
+  kube::test::get_object_assert pods "{{range.items}}{{$id_field}}:{{end}}" ''
+  # Command
+  file="${KUBE_TEMP}/redis-proxy-master.yaml"
```

Comment:
Sorry I didn't look closely enough first time. This works but it would be better to check in the full yaml file under examples. I sent you the fix to include/squash.